src/controllers/auth.js

```diff
@@ -0,0 +1,24 @@
+const User = require('../models/User');
```

Comment:
[Unknown Reference / Probably Git Mistake / Working Atomic Commits]
Your PR references this `../models/User` which does not exist on this branch.
I'm guessing this was a git mistake where you created this on a separate branch and incorrectly referenced it here.

So there's a few best practices in terms of how your git commits are supposed to be setup. One of these best practices is to use whats called "Working Atomic Commits". That's a technical way of saying any commit that you push should:
1. Work - ie do what its supposed to do or at least not break anything
2. Have exactly 1 new thing on it - not part of a thing and not multiple things

You actually did a pretty good job at atomic commits if we look commit by commit. There were a few commits that had changes to both the controller and changes to the package, but few and far between. No one does this exactly perfect either so pretty good. Keeping the commits small, frequent, and clear like this makes reviewers and coworkers/contributors jobs much easier. Which also means better feedback for you and less likelihood of breaking something by accident.

The working part of it is way more important though. And this code does not work 😢

Reply:
I will close this PR as I merged it with PRACT-16